_migration-assistant/migration-phases/backfill.md

```diff
@@ -139,17 +138,43 @@ console backfill scale 5
 
 We recommend slowly scaling up the fleet while monitoring the health metrics of the target cluster to avoid over-saturating it. [Amazon OpenSearch Service domains](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/monitoring.html) provide a number of metrics and logs that can provide this insight.
 
+### Pausing the migration
+
+To pause a migration, use the following command:
+
+```shell
+console backfill pause
+```
+
+This stop all existing workers from running while leaving the backfill operation in a state it can be restarted. When it's time to restart the migration, perform one of the following:
```

Comment:
```suggestion
This will stop all existing workers from running while leaving the backfill operation in a state from which it can be restarted. When you want to restart the migration, perform one of the following actions:
```